.circleci/config.yml

```diff
@@ -50,6 +50,9 @@ jobs:
       - run:
           name: Lint & Code styles
           command: yarn lint
+      - run:
+          name: Build
+          command: yarn build
```

Comment:
Why this change? It'll slow down CI. (Good to merge besides that.)

Reply:
The type-check step, which comes right after this line, is failing without this change. (see [circleci build](https://app.circleci.com/pipelines/github/algolia/react-instantsearch/1686/workflows/c2edf0ca-d57f-4309-bb86-0043bd7fbf98/jobs/48288))
The type-check step will check types in the examples, based off types inside `react-instantsearch-hooks`.
So `react-instantsearch-hooks` need to be built.